controller/toiletsController.js

```diff
@@ -1,4 +1,23 @@
-const { getReviews } = require("../service/toilets");
+const { getNearToilets, getReviews } = require("../service/toilets");
+
+exports.getNearToiletsList = async (req, res, next) => {
+  const { lat, lng } = req.query;
+  try {
+    const toiletList = await getNearToilets(lat, lng);
+
+    res.json({
+      result: "ok",
+      toiletList,
+    });
+
+    return;
+  } catch (error) {
+    res.json({
```

Comment:
실패한 경우 http 상태코드도 같이 넣어줘야 안전할 거같아요. 지금은 클라이언트에게 200으로 response가 넘어 가서 혼동이 올수 있을 것 같습니다. 

```suggestion
    res.status(400).json({
```